app/models/workflow_step.rb

```diff
@@ -6,7 +6,7 @@ class WorkflowStep < ApplicationRecord
   validates :workflow, presence: true
   validates :process, presence: true
   validates :version, numericality: { only_integer: true }
-  validates :status, inclusion: { in: %w[waiting started completed queued error skipped] }
+  validates :status, inclusion: { in: %w[waiting started completed queued error skipped retrying] }
```

Comment:
The status of a workflow step will be changed to "retrying" will the job is retrying.

Reply:
How is "retrying" set as a status? Is that to be implemented?

Reply:
Oh, I see, that's in lyber-core.